error[E0557]: feature has been removed
   --> /wrkdirs/usr/ports/www/firefox/work/firefox-93.0/third_party/rust/packed_simd/src/lib.rs:203:5
    |
203 |     const_generics,
    |     ^^^^^^^^^^^^^^ feature has been removed
    |
    = note: removed in favor of `#![feature(adt_const_params]` and `#![feature(generic_const_exprs)]`

https://github.com/rust-lang/packed_simd/commit/45d5347a0d2187c046a546a477d2a53111cd7713

--- third_party/rust/packed_simd/src/lib.rs.orig	2021-10-20 11:41:07 UTC
+++ third_party/rust/packed_simd/src/lib.rs
@@ -200,7 +200,7 @@
 //!   preserving, etc.
 
 #![feature(
-    const_generics,
+    adt_const_params,
     repr_simd,
     rustc_attrs,
     platform_intrinsics,
